lib/hexbin-view.js

```diff
@@ -36,7 +36,12 @@ export default class HexbinView {
           hex = '0x' + this.dec2hex(dec);
         }
         this.element.textContent = "Hex:" + hex + " Bin:" + this.formatBinary(bin) + " Dec:" + dec;
-    } else {
+      } else if (selection.length == 10 && selection.toUpperCase().substr(-10, 2) == '0B') {
```

Comment:
Can you please make the length variable instead of 10? i.e. a values "0b11011" should be valid. btw, line 28 "dec = parseInt(selection);" validates if the string is a number, if it fails the "if" is not entered, so you just need to check if te first 2 chars are "0b"